appveyor.yml

```diff
@@ -17,6 +17,7 @@ install:
   - git clone https://github.com/tadzik/rakudobrew %USERPROFILE%\rakudobrew
   - SET PATH=%USERPROFILE%\rakudobrew\bin;%PATH%
   - rakudobrew build moar %TEST_MOAR%
+  - SET PATH=%USERPROFILE%\rakudobrew\versions\moar-%TEST_MOAR%\install\bin:%PATH%

```

Comment:
This is the issues with appveyor failing I think should be 

SET PATH=%USERPROFILE%\rakudobrew\versions\moar-%TEST_MOAR%\install\bin;%PATH%

Reply:
Hahah yeah cheers.

Reply:
Well that definitely moved on the appveyor to a different fail in zef.

Reply:
Yeah. Might it be the Moar version?